backend/Controllers/PlayersController.cs

```diff
@@ -21,14 +20,17 @@ public PlayersController(IPlayersService playerService, ILogger<PlayersControlle
 			_logger = logger;
 		}
 
-		[HttpGet("{code?}")]
-		public IEnumerable<Player> Get([FromRoute] string code)
+		[HttpGet]
+		public async IAsyncEnumerable<Player> GetAll()
 		{
-			if (!String.IsNullOrEmpty(code)) {
-				return new[] { _playerService.Get(code)};
-			}
-
-			return _playerService.GetAll();
+			await foreach (var player in _playerService.GetAll())
+				yield return player;
+		}
+		
+		[HttpGet("{code}")]
+		public Task<Player> GetByCode([FromRoute] string code)
+		{
+			return _playerService.Get(code);
```

Comment:
I separated the GetByCode / GetAll as the return types are different. 
The returned models will probably differ as well soon, giving more detail when you load a single player.